src/resolvers.ts

```diff
@@ -212,6 +213,66 @@ const resolvers = {
             return createSubmissionResult.data
         },
 
+        moderationPanelUpdateSubmission: async (_parent: unknown, args: {
```

Comment:
this name doesn't make sense to me. What is the action being performed that's different from `updateSubmission`? I would either make this a subset of functionality under the `update` resolver or make one that's more specific. (`approveSubmission` is a good example of one that's part of updateSubmission. It could technically be a separate resolver, but realistically it's under update because it's an update to the status of submission that has side effects. This feels like a similar case to me)

Reply:
OK

Reply:
Makes sense to me